### PR TITLE
Book store1

### PR DESCRIPTION
**Pull request by arikarim**
Project setup for milestone 1 of Book-store project:

- [x] Setup Linters
- [x] Add Redux and React-Redux
- [x] Add Proptypes
- [x] Deployed to heroku

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "import/no-extraneous-dependencies": "off"
```
Comment (bellom) [inline .eslintrc.json]: We have some little changes in `.eslintrc.json`,  check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux), according to the project requirement we need to set the linter's correctly with the standard Microverse config file without changing it.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @arikarim  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: .eslintrc.json (hunk 1 above)
Comment (arikarim) [reply .eslintrc.json]: it another file from a previous project I added accidentally, 
thanks for the note.
Done :+1:

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @arikarim**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**